hooks/src/index.css

```
@@ -1,68 +1,133 @@
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  font-family: sans-serif !important;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Biome (1.9.4)

[error] Unexpected shorthand property font after font-family



(lint/suspicious/noShorthandPropertyOverrides)